# Header
# frame id here is destination
std_msgs/Header header

# source frame id
string src_frame_id

# Pillar Pose
# Distance
int64 distance
float64 angle
